charts/k8ssandra/templates/stargate/stargate.yaml

```diff
@@ -111,11 +111,13 @@ spec:
               path: /checker/liveness
               port: health
             timeoutSeconds: 10
-            initialDelaySeconds: 240
+            failureThreshold: 10
```

Comment:
Why are these modified?

Reply:
yup, my bad, it's a leftover from some other work I was doing.
Good catch 👍

Reply:
Removed 👌